internal/command/machine/clone.go

```diff
@@ -262,7 +262,15 @@ func getAppConfig(ctx context.Context, appName string) (*appconfig.Config, error
 		return cfg, nil
 	}
 
-	err, _ := cfg.Validate(ctx)
+	client := client.FromContext(ctx)
+	appBasic, err := client.API().GetAppBasic(ctx, appName)
+	if err != nil {
+		return nil, err
+	}
+
+	cfg.AppName = appBasic.Name
```

Comment:
Not sure we need this and it is an extra roundtrip to get the application details from Web API. 
If we want to be sure cfg returned by FromRemoteApp has a name, so set it there inside the function where it already has all the context of the requested app name